Runtime/Plugins/WebGL/VideoRenderer.jslib

```diff
@@ -0,0 +1,58 @@
+var UnityWebRTCVideoRenderer = {
+  CreateVideoRenderer: function (contextPtr) {
+  },
+
+  CreateNativeTexture: function() {
+    //console.log('nativeTexture');
+    var texPtr = 0;
+    for(var texPtr = 0; texPtr < GL.textures.length; texPtr++) {
+      if(GL.textures[texPtr] === undefined)
+        break;
+    }
+    var tex = GLctx.createTexture();
+    tex.name = texPtr;
+    GL.textures[texPtr] = tex;
+    //console.log('nativeTexture' + texPtr);
+    return texPtr;
+  },
+  
+  GetVideoRendererId: function (sinkPtr) {
+  },
+
+  DeleteVideoRenderer: function (contextPtr, sinkPtr) {
+
+  },
+
+  UpdateRendererTexture: function (trackPtr, renderTexturePtr, needFlip) {
```

Comment:
The `needFlip` argument is not used in this method.
I would like to remove this argument.

Reply:
Sure, will remove it! 

I kept it in place since I was fighting a bit with the default flipping of unity. In theory you could _not_ flip manually, so an y-inverted image is send over webrtc and have it (due to default flipping) correctly shown on the receiver.
But this becomes a problem when using a webcam mediastream, as they should be manually flipped on the receiver because that image is not y-inverted on the webrtc stream. So in order to send the correct image over webrtc, I always flip.

Reply:
> But this becomes a problem when using a webcam mediastream, as they should be manually flipped on the receiver because that image is not y-inverted on the webrtc stream. So in order to send the correct image over webrtc, I always flip.

Certainly, This is the problem. Should we add the API to control whether flip the streaming video?

Reply:
I'm confused, do you want to be able to flip the image or not, both receiver/sender? 
At least on the receiving side I would always flip, as we should be expect the incoming stream to be correct and not flipped.
As a sender, it might be good to have the flip option I guess, as unity flips it but webcam stream doesn't. I'm not sure if there are other cases where unity webgl sends the non-flipped image.